.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
```

Comment:
https://github.com/andynarf/library/pull/1#discussion_r372239324
Please let us make use of the Microverse config files 👍.